Example/Podfile

```diff
@@ -5,7 +5,7 @@ platform :ios, '9.0'
 use_frameworks!
 
 def project_pods
-    pod 'Gini-iOS-SDK'
+    pod 'Gini-iOS-SDK', :git => 'git@github.com:gini/gini-sdk-ios.git' ,:tag => '1.2.0-beta.1'
```

Comment:
Is the git URL actually needed?

Reply:
I thought it wasn't but didn't work for me locally

Reply:
Also, you should be using the HTTPS URL not the git one.